tests/e2e/e2e_deployment_client_server_test.go

```diff
@@ -57,13 +65,28 @@ var _ = OSMDescribe("Test HTTP traffic from N deployment client -> 1 deployment
 						Name:         "server",
 						Namespace:    destApp,
 						ReplicaCount: int32(replicaSetPerService),
-						Image:        "kennethreitz/httpbin",
+						Image:        "simonkowallik/httpbin",
```

Comment:
Why change the httpbin image?

Reply:
The kennethreitz httpbin app does not take the passed in `XHTTPBIN_<headername>` env var, thus we cannot use it to distinguish the pod name.